### ISSUE TITLE
Availability view of other people

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/server#44776]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

## Feature Request

As a Nextcloud user of an organization that does their daily work on Nextcloud I often miss a quick way to see if a colleague is available, or when they are back in case of unavailability. My current workaround is to start a new event in Calendar, add the person as attendee and open the free/busy view. Assuming that people do maintain their general availability and use the out of office feature for absence, this gives me a reliable insight.

I wish this same information was accessible more easily. The problem becomes more visible with colleagues who work part time and teams distributed across time zones.

## Implementation

Add a UI component to render the availability of one single user. We might be able to just use the existing free/busy feature for this.
This UI could show at the profile, for example, or in the Calendar app.

## Work packages
- [ ] Implementation
- [ ] Admin manual documentation (if applicable, e.g. to turn the feature off)
- [ ] User manual documentation

**Comment by jancborchardt** [comment]
Some ideas to make this easier:
- For an individual person, "See availability" should be an action in the avatar menu. This would just open the free/busy modal with availability of both you and the person.
- In calendar, there could be an entry in the left sidebar like "Team availability" which takes the team info from Contacts and/or account mgmt and opens the free/busy modal with you and all of the people in your team.
- Additionally, it would be nice if you can add arbitrary people in this modal to check their availability, e.g. if you need them for a call, or for support, etc.

In all cases it would use the existing free/busy modal so we don’t need to have so much new interface for it.

**Comment by jancborchardt** [comment]
@nimishavijay do you have additional ideas? :) And supporting mockups for this would be very good, as it’s part of this issue https://github.com/nextcloud/server/issues/43763

**Comment by marcoambrosini** [comment]
This was picked in planning, Nimisha could you start working on it?

**Comment by Mer0me** [comment]
It would be nice to be able to configure automatic status for remote work, too. And maybe a specific calendar category for this particular position : you actually work, but you cannot be present for a real meeting in an office room. I don't know if caldav protocol can handle this kind of precision.

Nextcloud could detect keywords ("Home work / remote work... don't forget translations too !)  in calendar meetings and set the status accordingly. Alternatively it can be a specific category on the availability page ?

**Comment by alexanderdd** [comment]
related? https://github.com/nextcloud/server/issues/37815

**Comment by st3iny** [comment]
Yeah, seems related. We discussed showing the availability on a profile but deprioritized it for now. The profile is meant to be a public/external thing and the availability should be shown internally.

**Comment by jancborchardt** [comment]
> For an individual person, "See availability" should be an action in the avatar menu. This would just open the free/busy modal with availability of both you and the person.

As per discussion with @ChristophWurst, this approach is what makes most sense as it is available everywhere in Nextcloud.

**Comment by st3iny** [comment]
Is done. All three PRs have been merged.

**Comment by ChristophWurst** [comment]
@st3iny could this feature benefit from some user or admin docs?
